### DISCUSSION TITLE
How to use the routesFolder option

### DISCUSSION THREAD

**Discussion by tongcheong** [posva/unplugin-vue-router#300] [Questions and Help] [answered]
Hi, I understand from the README that this vue router plugin scans the 'src/pages' folder, may I know how can I change this configuration so that it scans other folders i.e. 'src/views' ?

**Comment by posva** [comment]
You just put that path instead of src/pages https://github.com/posva/unplugin-vue-router#configuration

**Reply by tongcheong** [reply]
Hi, my apologies I'm new to this. How do I access the VueRouter class ? Where do I import it ?

**Reply by posva** [reply, marked as answer]
It depends on what you use, for Vite, it's

```ts
// vite.config.ts
import VueRouter from 'unplugin-vue-router/vite'

export default defineConfig({
  plugins: [
    VueRouter({
      /* options */
    }),
    // ⚠️ Vue must be placed after VueRouter()
    Vue(),
  ],
})
```

You have the other examples in the readme, collapsed.

**Reply by tongcheong** [reply]
Thanks for the reply, did read the README but couldnt figure it out. Will try it out.
